client/pool.c

```diff
@@ -1347,9 +1347,10 @@ int pay_miners(xdag_time_t time)
 
 	double *diff = malloc(2 * miners_count * sizeof(double));
 	if(!diff) return -8;
-	double *prev_diff = diff + miners_count;
 
-	if(!precalculate_payments(hash, confirmation_index, &data, diff, prev_diff, nonce)) {
+	double *prev_diff = diff + miners_count;
+	double prev_sum = precalculate_payments(hash, confirmation_index, &data, diff, prev_diff, nonce);
+	if(prev_sum == 0) {
```

Comment:
Here I have a concern about double value == 0. EPSILON would be much more precise for this case. `double value <= EPSILON.`

Reply:
Let I check at evening if VS has definition for EPSILON

Reply:
should be DBL_EPSILON not just EPSILON  and it is in c standard.. you should have it too in float.h